test/30_debate/clientfunctions.js

```diff
@@ -366,7 +366,7 @@ describe("Debate client functions", () => {
             });
 
             client.emit('suggestQuestion', 'This is my new suggestion.', res => {
-                res.should.equal(true);
+                res.should.not.equal(true);
```

Comment:
Same comment.

Reply:
suggestQuestion was modified to return false on error and the suggestion id on success (it is a necessary modification for the client app).
res.should.not.equal(false) allows me to verify that there is no error, rather than checking for a particular id.

Reply:
Fixed it in 8512092.